lib/src/Uncertainty/Distribution/openturns/ConditionalDistribution.hxx

```diff
@@ -1,171 +1 @@
-  PointWithDescriptionCollection getParametersCollection() const override;
-
-  /** Parameters value accessors */
-  void setParameter(const Point & parameter) override;
-  Point getParameter() const override;
-
-  /** Parameters description accessor */
-  Description getParameterDescription() const override;
-
-  /* Interface specific to ConditionalDistribution */
-
-  /** Conditioned distribution accessor */
-  void setConditionedDistribution(const Distribution & conditionedDistribution);
-  Distribution getConditionedDistribution() const;
-
-  /** Conditioning distribution accessor */
-  void setConditioningDistribution(const Distribution & conditioningDistribution);
-  Distribution getConditioningDistribution() const;
-
-  /** Link function accessor */
-  void setLinkFunction(const Function & linkFunction);
-  Function getLinkFunction() const;
-
-  /** Get the i-th marginal distribution */
-  Distribution getMarginal(const UnsignedInteger i) const override;
-
-  /** Get the distribution of the marginal distribution corresponding to indices dimensions */
-  Distribution getMarginal(const Indices & indices) const override;
-
-  /** Method save() stores the object through the StorageManager */
-  void save(Advocate & adv) const override;
-
-  /** Method load() reloads the object from the StorageManager */
-  void load(Advocate & adv) override;
-
-
-protected:
-
-
-private:
-
-  /** set both parameters */
-  void setConditionedAndConditioningDistributionsAndLinkFunction(const Distribution & conditionedDistribution,
-      const Distribution & conditioningDistribution,
-      const Function & linkFunction);
-
-  /** Compute the expectation of f(\theta)1_{\theta\leq \theta^*} with respect to the prior distribution of \theta */
-  Point computeExpectation(const Function & f,
-                           const Point & thetaStar) const;
-
-  /** The conditioned distribution, i.e L(X|Theta) */
-  Distribution conditionedDistribution_;
-
-  /** The conditioning distribution, i.e L(Theta) */
-  Distribution conditioningDistribution_;
-
-  /** The link function */
-  Function linkFunction_;
-
-  /** Discrete marginals indices */
-  Indices discreteMarginalsIndices_;
-
-  /** Dirac marginals indices */
-  Indices diracMarginalsIndices_;
-
-  /** Continuous marginals indices */
-  Indices continuousMarginalsIndices_;
-
-  /** Lower bounds of the continuous marginals */
-  Point continuousLowerBounds_;
-
-  /** Upper bounds of the continuous marginals */
-  Point continuousUpperBounds_;
-
-  /** Standard continuous integration nodes */
-  Sample continuousNodes_;
-
-  /** Standard continuous weights */
-  Point continuousWeights_;
-
-  /** Discrete integration nodes */
-  Sample discreteNodes_;
-
-  /** Values of the Dirac marginals */
-  Point diracValues_;
-
-}; /* class ConditionalDistribution */
-
-
-END_NAMESPACE_OPENTURNS
-
-#endif /* OPENTURNS_CONDITIONALDISTRIBUTION_HXX */
+#include "openturns/DeconditionedDistribution.hxx"
```

Comment:
You can drop it completely from imho, there should not be much users of this in c++

Reply:
I remove ConditionalDistribution.hxx from the source tree? No consequence on the SWIG interface?

Reply:
you can drop it as its available from DeconditionedDistribution.hxx

Reply:
done